mesop/server/server.py

```diff
@@ -49,6 +51,8 @@
 
 STREAM_END = "data: <stream_end>\n\n"
 
+DEFAULT_ASSETS_URL_PATH = "/assets"
```

Comment:
I think this is a reasonable default value, but we should probably also make it configurable  (e.g. they may already have a page with the path "/assets")

Reply:
Yup. I agree that this should be configurable.